tools/online_editor/src/properties_widget.ts

```diff
@@ -120,6 +163,18 @@ export class PropertiesWidget extends Widget {
         value_field.innerText = "";
       }
       row.appendChild(value_field);
+      console.log("What do we have here:", p);
```

Comment:
```suggestion
```

Reply:
> I want to rebase this, so I updated the original commit.

Yes, in general, when i put suggestion, please do whatever is the most convenient for you to add that to your patch set.  You can do it manually locally, or apply through github, it doesn't matter.  What matters is that the console.log don't stay :-P